components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -157,6 +157,7 @@ input ProvisionRuntimeInput {
 
 input ClusterConfigInput {
     gardenerConfig: GardenerConfigInput!     # Gardener-specific configuration for the cluster to be provisioned
+    administrators: [String]                # List of administrators
```

Comment:
is this field can be empty?

Reply:
it's a protection against the situation when during upgrade `Provisioner` run before `KEB`.  In this situation `KEB` doesn't send this param.